src/aics-image-viewer/components/ViewerStateProvider/types.ts

```diff
@@ -66,6 +68,10 @@ export interface ChannelState {
   ramp: [number, number];
   useControlPoints: boolean;
   controlPoints: ControlPoint[];
+  /** Internal only. If set, flags that this channel should be reset to the default LUT when the
+   * next volume is loaded or when the viewer is updated.
+   */
+  needsDefaultLut?: boolean;
```

Comment:
Had to add this to make the "Clear all settings" button work. (Alternatively I also could have overloaded the control points so `controlPoints=[]` indicates that it should be set to the defaults, but I worried that this might be too ambiguous/packing extra non-intuitive behavior.) Open to feedback here.